src/ImageSharp/Formats/Jpeg/JpegEncoder.cs

```diff
@@ -1,25 +1,28 @@
 // Copyright (c) Six Labors.
 // Licensed under the Six Labors Split License.
 
-using SixLabors.ImageSharp.PixelFormats;
-
 namespace SixLabors.ImageSharp.Formats.Jpeg;
 
 /// <summary>
 /// Encoder for writing the data image to a stream in jpeg format.
 /// </summary>
-public sealed class JpegEncoder : IImageEncoder, IJpegEncoderOptions
+public sealed class JpegEncoder : ImageEncoder
 {
     /// <summary>
     /// Backing field for <see cref="Quality"/>.
     /// </summary>
     private int? quality;
 
-    /// <inheritdoc/>
+    /// <summary>
+    /// Gets the quality, that will be used to encode the image. Quality
+    /// index must be between 0 and 100 (compression from max to min).
```

Comment:
Should be between 1 and 100 as code itself throws on 0 :)

Maybe it's also wise to 'promote' [1..99] range in remarks as jpeg spec doesn't advise quality 100 it doesn't really preserve 'quality' and can lead to artifacts. Most likely I'll just bump this up in some of the PRs when I'll have time to actually commit.

Reply:
I didn't know that about quality 100! Weird that it does that. I wonder if we should simply clamp to 1-99?

Reply:
I don't think so, quality=100 is still technically correct according to the spec. I believe regular users treat quality as percent value, would be weird to be clamped at 99% from user perspective :)